src/cladetime/util/reference.py

```diff
@@ -70,7 +70,9 @@ def _get_s3_object_url(bucket_name: str, object_key: str, date: datetime) -> Tup
     return version_id, version_url
 
 
-def _run_nextclade_cli(nextclade_cli_version: str, nextclade_command: list[str], output_file: Path) -> Path:
+def _run_nextclade_cli(
+    nextclade_cli_version: str, nextclade_command: list[str], output_file: Path, input_files: list[Path] | None = None
```

Comment:
expand function signature to enable nextclade cli commands that require input files (i.e., we need to mount those files to the nextclade cli docker container)